subsys/caf/modules/Kconfig.ble_adv

```diff
@@ -35,6 +35,62 @@ config CAF_BLE_ADV_DIRECT_ADV
 config CAF_BLE_ADV_FAST_ADV
 	bool "Enable period of fast advertising"
 
+config CAF_BLE_ADV_SLOW_INT_MIN
+	hex "Minimum Advertising Interval during slow advertising (N * 0.625 milliseconds)"
+	range 0x0020 0x4000
+	default 0x00a0 # 100 ms
+	help
+	  Minimum Advertising Interval shall be less than or equal to the
+	  Maximum Advertising Interval. The Minimum Advertising Interval and
+	  Maximum Advertising Interval should not be the same value (as stated
+	  in Bluetooth Core Spec 5.2, section 7.8.5)
+
+config CAF_BLE_ADV_SLOW_INT_MAX
+	hex "Maximum Advertising Interval during slow advertising (N * 0.625 milliseconds)"
+	range 0x0020 0x4000
+	default 0x00f0 # 150 ms
+	help
+	  Minimum Advertising Interval shall be less than or equal to the
+	  Maximum Advertising Interval. The Minimum Advertising Interval and
+	  Maximum Advertising Interval should not be the same value (as stated
+	  in Bluetooth Core Spec 5.2, section 7.8.5)
+
+if CAF_BLE_ADV_FAST_ADV
+
+config CAF_BLE_ADV_FAST_INT_MIN
+	hex "Minimum Advertising Interval during fast advertising (N * 0.625 milliseconds)"
+	range 0x0020 0x4000
+	default 0x0030 # 30 ms
+	help
+	  Minimum Advertising Interval shall be less than or equal to the
+	  Maximum Advertising Interval. The Minimum Advertising Interval and
+	  Maximum Advertising Interval should not be the same value (as stated
+	  in Bluetooth Core Spec 5.2, section 7.8.5)
+
+config CAF_BLE_ADV_FAST_INT_MAX
+	hex "Maximum Advertising Interval during fast advertising (N * 0.625 milliseconds)"
+	range 0x0020 0x4000
+	default 0x0060 # 60 ms
+	help
+	  Minimum Advertising Interval shall be less than or equal to the
+	  Maximum Advertising Interval. The Minimum Advertising Interval and
+	  Maximum Advertising Interval should not be the same value (as stated
+	  in Bluetooth Core Spec 5.2, section 7.8.5)
+
+endif
+
+if !CAF_BLE_ADV_FAST_ADV
+
+config CAF_BLE_ADV_FAST_INT_MIN
```

Comment:
Please make it consistent with handling default values for other Kconfig options in ble_adv.